.env.example

```diff
@@ -1,18 +1,26 @@
-# local, rinkeby, goerli, etc
+# rinkeby, trufflepig
+LOADER=trufflepig
+
+# local, goerli, mainnet
 NETWORK=local
-CHAIN_ID=1337
 
-# trufflepig, network
-LOADER=network
+# chain id
+# local - 1337
+# goerli - 5
+# mainnet - 1
+CHAIN_ID=1337
 
-# .eth for main/devnet, .test for testnet
-COLONY_NETWORK_ENS_NAME=joincolony.eth
+# pinning service pubsub room
+PINNING_ROOM=PINION_DEV_ROOM
 
-SKIP_GAS_STATION_CONFIRM=true
+# verbose logger
 VERBOSE=true
 
-# API keys for development
+# ethplorer key
 ETHPLORER_API_KEY=freekey
 
-# Infura ID
-INFURA_ID=blahblah
+# ENS suffix
+COLONY_NETWORK_ENS_NAME=joincolony.eth
```

Comment:
Could benefit from similar explanation to `CHAIN_ID`. `.eth` is used for local and mainnet, `.test` used for goerli.

Reply:
Will add